.github/workflows/ci.yml

```diff
@@ -189,17 +189,23 @@ jobs:
 
   test-stable:
     name: cargo test
-    runs-on: ${{ matrix.os }}
+    runs-on: ${{ matrix.platform.os }}
     strategy:
       matrix:
-        os: [windows-latest, macos-latest, ubuntu-latest]
+        platform:
+          - { target: aarch64-apple-darwin, os: macos-14 }
```

Comment:
Why not macos-15 or macos-latest?

Reply:
Sure, but not for too much longer.

Reply:
> Sure, but not for too much longer.

Indeed, but in servo we prefer to hardcode runner images, because updates usually broke something, at least in more complex (ahm C++) libraries we have.